### DISCUSSION TITLE
Preservation-toiminnon käännös?

### DISCUSSION THREAD

**Discussion by AnneliO** [KohaSuomi/Koha-translations#29] [Yleinen]
23.11-versiossa on myös Preservation-niminen uusi toiminto: https://bugs.koha-community.org/bugzilla3/show_bug.cgi?id=30708

Yritin tuota testata ja ymmärtää ja jos oikein ymmärrän, niin sillä voi tehdä jonkinlaisia "käsittelyjonoja" tai "eriä" (trains) esim. hankinnassa oleville niteille. Eli vaikkapa muovitusta odottavat niteet.

Toiminnossa on esim. tällaisia termejä:
* Preservation
* Preservation train
* Trains
* New train
* Waiting list

Olisko näille ideoita?

**Comment by AnneliO** [comment]
Nyt toiminnon kuvaus näyttäis löytyvän [manuaalistakin](https://koha-community.org/manual/latest/en/html/preservation.html#).

**Reply by AnneliO** [reply]
Manuaalin esimerkkinä on sidottavaksi laitettavat niteet.

**Comment by AnneliO** [comment]
*Preservation*-sana itsessään suoraan käännettynä on esim. säilyttäminen, suojeleminen, säilytys, säilöntä, varjeleminen, varjelu, suojelu

*Trains*: The term “train” has been chosen for two main reasons. Firstly because all of the other terminology ‘cart’, ‘basket’, batch, etc. were already used for other modules in Koha. Secondly, this is a commonly used terminology by French librarians, since the physical carts aligned look like linked wagons.

*Waiting list*: Odotuslista?

**Reply by AnneliO** [reply]
Voisiko Train olla kuitenkin "erä", vaikka ominaisuuden tekijät ovatkin yrittäneet sitä välttää. Minusta se olisi sinänsä johdonmukainen nimi, kun käsitellään tiettyä joukkoa niteitä.

**Reply by inkerih** [reply]
Onko meillä millaisia kännöksiä noille jo aiemmin käytössä olleille "erille", siis esim. ‘cart’, ‘basket’, batch ? Onko niiden suomennoksessa mitään johdonmukaisuutta?

**Reply by AnneliO** [reply]
cart = kori
basket = tilauskori
batch = erä tai eräajo vähän lauseyhteydestä riippuen

Uskoisin noiden olevan aika johdonmukaisesti noin, koska kaikki suomennokset on käyty projektina läpi ja sen jälkeen on yritetty noudattaa samoja termejä kuin aiemmin (tai siis ainakin itse olen yrittänyt noudattaa).

**Reply by inkerih** [reply]
Onko "käsittelyerä" liian kömpelö? Mutta siitä olen samaa mieltä, että juniksi näitä ei voi kääntää. Eikä vaunutkaan kuulosta järkeviltä.

**Reply by AnneliO** [reply]
Minusta käsittelyerä olisi ihan toimiva termi tähän yhteyteen.

**Reply by inkerih** [reply]
Ainiin, ja kirjastomaailmassa preservation voi olla myös konservointi. Mutta esim. pelkkään kirjojen muovittamiseen tms. se minusta istuu huonosti. Mutta en nyt äkkiseltään keksi, mikä noista suorista suomennoksista tuolle sanalle olisi paras.

Waiting list minusta voisi ihan hyvin olla tuo ehdotettu odotuslista.

**Reply by AnneliO** [reply]
Saisko tuosta käsittelyerä-sanasta johdettua jonkin nimen?

**Reply by inkerih** [reply]
Säilytyksen käsittelyerät? Konservoinnin käsittelyerät? Säilytystoimenpiteet? Käsittelyerät?

**Reply by AnneliO** [reply]
Melkein kyllä haluttais ehdottaa tuota Käsittelyerää. Vaikka tulisikin sekä preservation- ja train-sanoille sama käännös. (Itsepähän ovat valinneet noin hassut termit x))

**Reply by inkerih** [reply]
Joo. Tässä on minusta hassua, että tällainen jokseenkin yleisluontoinen toiminto (nide-erien käsittely) on nimetty noin spesifillä termillä kuin "preservation". Sen vähän perusteella, mitä tähän toimintoon olen ehtinyt perehtyä, näkisin, että tätä voi käyttää aika monenlaiseenkin juttuun. Meillä yksi käyttötarkoitus voisi olla esim. digitointi. Digitoitavaksi otetaan kerralla isompi erä, ja ne ovat sitten pois lainakäytöstä sen aikana, ja mahdollisesti sen jälkeenkin niihin laitetaan jotain tilatietoja, huomautuksia ym. Eli tän toiminnon yleisluontoisuuden vuoksi tuo "Käsittelyerä" voisi olla ihan hyvä termi tälle.

**Reply by AnneliO** [reply]
Miepä käännän nuo nyt tuolla Käsittelyerät-termillä ja jos se alkaa ajan kuluessa tuntumaan huonolta, niin vaihdetaan sitten. Waiting list sitten odotuslistaksi. :)

**Comment by pknuuti** [comment]
Kuulostaa järkevältä. Suora käännös ”juna” on lähinnä omituinen, sitä ei ainakaan voi käyttää. 😊

Päivi Knuutinen
kirjastovirkailija
Vaara-kirjastot
050 441 2247
***@***.***

Lähettäjä: Anneli Österman ***@***.***>
Lähetetty: 4. joulukuuta 2023 15:21
Vastaanottaja: KohaSuomi/Koha-translations ***@***.***>
Kopio: Subscribed ***@***.***>
Aihe: Re: [KohaSuomi/Koha-translations] Preservation-toiminnon käännös? (Discussion #29)


Voisiko Train olla kuitenkin "erä", vaikka ominaisuuden tekijät ovatkin yrittäneet sitä välttää. Minusta se olisi sinänsä johdonmukainen nimi, kun käsitellään tiettyä joukkoa niteitä.
